### PR TITLE
refactor(images): use operator-framework namespace for all olm images

### PR DESCRIPTION
**Pull request by njhale**
Add a few changes around upstream images and packaging:
- Use OLM images for all OLM images
- Use existing manifests in `package_quickstart.sh`
- Add quickstart generation to make `release` rule
- Build with go 1.12 in `upstream.Dockerfile`

Addresses part of #747 and [ALM-937](https://jira.coreos.com/browse/ALM-937).

### CODE REVIEW

File: scripts/build_local.sh [hunk 1]
```diff
@@ -10,6 +10,6 @@ if [ -z "$NO_MINIKUBE" ]; then
   eval $(minikube docker-env) || { echo 'Cannot switch to minikube docker'; exit 1; }
   kubectl config use-context minikube
 fi
-docker build -f e2e.Dockerfile .
-docker tag $(docker images --filter 'label=stage=olm' --format '{{.CreatedAt}}\t{{.ID}}' | sort -nr | head -n 1 | cut -f2) quay.io/coreos/olm:local
-docker tag $(docker images --filter 'label=stage=builder' --format '{{.CreatedAt}}\t{{.ID}}' | sort -nr | head -n 1 | cut -f2) quay.io/coreos/olm-e2e:local
+docker build -f upstream.Dockerfile .
```
Comment (ecordell) [inline scripts/build_local.sh]: Why the switch to upstream here?

Comment (njhale) [reply scripts/build_local.sh]: I thought it would be nice to use the same image for local testing - I can put it back if you'd like.

**Comment by alecmerdler** [comment]
Note that we are using an image in `operatorframework` in the [console end-to-end tests for OLM](https://github.com/openshift/console/blob/master/frontend/integration-tests/tests/olm/catalog.scenario.ts#L21) (to add a `CatalogSource` without Marketplace). I pushed the same image to the `operator-framework` org, but haven't made a PR against `openshift/console` to use it yet. Hopefully no one deletes that organization.

**Comment by ecordell** [comment]
@njhale looks like a rebase is in order

**Comment by njhale** [comment]
> Hopefully no one deletes that organization.

@alecmerdler we're using those images in OLM's e2e tests as well. I was planning on opening a PR with the `upstream.Dockerfile` for `operator-registry` today and rigging the Quay build-triggers.

**Pull request merged**